### PR TITLE
fix: handle 2-bytes to 6-bytes unicode characters

### PR DESCRIPTION
**Pull request by ArTiSTiX**
Fixes https://github.com/patrickhulce/fontmin-webpack/issues/54

Allow 2 to 6 bytes unicode characters to be recognized as glyphs.

### CODE REVIEW

File: test/fixtures/entry.css [hunk 1]
```diff
@@ -30,6 +30,10 @@
   content: '\f0ce';
 }
 
+.custom-content {
+  content: '\f0'; /* ð */
```
Comment (patrickhulce) [inline test/fixtures/entry.css:34]: shouldn't we assert that the font contains this in our tests?

**Review by patrickhulce — COMMENTED** [review]
test would be much appreciated here :)

File: test/fixtures/entry.css (hunk 1 above)
Comment (ArTiSTiX) [reply test/fixtures/entry.css:34]: I added this to ensure it does not match all "\f0xx" unicodes from font icons.

**Comment by ArTiSTiX** [comment]
@patrickhulce I tried to implement tests with font-awesome 6, but unfortunately, that's the only icon pack i know that uses 2-bytes glyphs, and since version 6, they no longer provide the SVG webfont used in test to check glyphset is correct.

I could add a custom pack generated by icomoon though...

**Comment by ArTiSTiX** [comment]
@patrickhulce Thanks for inciting me to add the test, as i had to fix something.

I added a custom-font generated from icomoon.
Unfortunately, Icomoon does not put names on the TTF glyphs, so i had to convert the svg font back to ttf in FontForge (so it's not the original output from icomoon, even though it should work fine with the original ttf icomoon provides).

**Review by patrickhulce — APPROVED** [review]
thank you!

**Comment by ArTiSTiX** [comment]
@patrickhulce Thanks for all the merges.

Are you planning to publish as 4.0.0 ?

**Comment by patrickhulce** [comment]
yep it's published @ArTiSTiX thanks for all your help!

**Comment by ArTiSTiX** [comment]
Thanks a lot @patrickhulce !

**Pull request merged**